nominal/core/client.py

```diff
@@ -442,6 +444,48 @@ def get_commensurable_units(self, unit_symbol: str) -> Sequence[scout_units_api.
         """Get the list of units that are commensurable (convertible to/from) the given unit symbol"""
         return self._clients.units.get_commensurable_units(self._clients.auth_header, unit_symbol)
 
+    def get_channel(self, rid: str) -> Channel:
+        """Get metadata for a given logical series by looking up its rid
+        Args:
+            rid: Identifier for the logical series to look up
+        Returns:
+            Resolved metadata for the requested logical series
+        Raises:
+            conjure_python_client.ConjureHTTPError: An error occurred while looking up the logical series.
+                This typically occurs when there is no such logical series for the given RID.
+        """
+        return Channel._from_conjure(self._clients.logical_series.get_logical_series(self._clients.auth_header, rid))
+
+    def set_channel_units(self, rids_to_types: Mapping[str, str | None]) -> Sequence[Channel]:
+        """Sets the units for a set of channels based on user-provided unit symbols
+        Args:
+            rids_to_types: Mapping of channel RIDs -> unit symbols (e.g. 'm/s').
+                NOTE: Providing `None` as the unit symbol clears any existing units for the channels.
+        Returns:
+            A sequence of metadata for all updated channels
+        Raises:
+            conjure_python_client.ConjureHTTPError: An error occurred while setting metadata on the channel.
+                This typically occurs when either the units are invalid, or there are no
+                channels with the given RIDs present.
+        """
+        series_updates = []
+        for rid, series_type in rids_to_types.items():
+            if series_type is None:
+                unit_update = timeseries_logicalseries_api.UnitUpdate(clear_unit=timeseries_logicalseries_api.Empty())
+            else:
+                unit_update = timeseries_logicalseries_api.UnitUpdate(unit=series_type)
```

Comment:
lets move this to a function - generally want to avoid conditionally declaring variables, and a function which just returns helps remove that

Reply:
think you used this in one place but not all :)